### ISSUE TITLE
The selected and the active list items have the same styles applied

### ISSUE THREAD

**Issue by dafo** [IgniteUI/ignite-ui#810]
### Description  
The item that is selected has the same color as the one that is just active. It would be good to distinguish those states.

### Steps to reproduce  

1.
```
$('#editor').igNumericEditor({
	buttonType: "dropdown",
	listItems: [1, 4, 5]
});
```
2. Open the dropdown list
3. Click '1'
4. Open the list again
5. Press the DOWN arrow key

### Result  
Botsh items '1' and '4' have the same look

### Expected result  
The color of the active item shouldn`t be the same as the selected one.

### Attachments  
![screenshot](https://cloud.githubusercontent.com/assets/2778393/22647390/de376478-ec79-11e6-835b-87af1707d03c.png)

**Comment by damyanpetev** [comment]
The issue is that both used styles (`.ui-igedit-listitemselected.ui-state-highlight` and `.ui-state-active.ui-igedit-listitemactive`) set the same colors in the themes defined as rules for `.ui-igedit-dropdown .ui-igedit-listitem.ui-state-highlight` and `.ui-igedit-dropdown .ui-igedit-listitem.ui-state-active` respectively.

> NB! This seems to apply to all themes.

I think it's only logical to follow the same color pattern as the `igCombo` ([sample](https://www.igniteui.com/combo/selection-and-checkboxes)):
![image](https://cloud.githubusercontent.com/assets/3198469/25334416/3993b15a-28f7-11e7-9e31-57fbd9ea5fb8.png)
Where the active item (`ui-igcombo-item-in-focus`) is a lighter tone and also overrides the selected style when navigating over existing items. It's a separate class but i think moving the existing editor selector to the same rule would be better, to avoid changing the classes we use.

@IG-Stanimir-Dimitrov @bazal4o Thoughts?

**Comment by IG-Stanimir-Dimitrov** [comment]
I am ok having a lighter toned styling class for the list active item.

**Comment by IG-Stanimir-Dimitrov** [comment]
@desig9stein ,can you please change the color of active dropdown item for the editors [here](https://github.com/IgniteUI/ignite-ui/blob/master/src/css/structure/modules/infragistics.ui.editors.css) with a lighter tone similar to the combo styling?

**Comment by IG-Stanimir-Dimitrov** [comment]
@desig9stein , @kdinev has merged the changes into the [#810 branch](https://github.com/IgniteUI/ignite-ui/tree/%23810), not the Ignite UI master branch. Can you please open another pull request to add them to the [master](https://github.com/IgniteUI/ignite-ui)?
